js/foam/u2/Element.js

```diff
@@ -830,6 +873,7 @@ CLASS({
 
       var first = true;
       for ( var key in this.classes ) {
+        if ( ! this.classes[key] ) continue;
```

Comment:
If a class is added using `enableCls('some-class', someValue$, true /* negate */)` then we have `this.classes['some-class'] === false`; but without this check the class was emitted anyway.

Removing classes was working dynamically, but not if they're disabled at output time.